aether-producer/producer/__init__.py

```diff
@@ -15,3 +15,644 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
```

Comment:
You can ignore this file. The reason it shown as 100% diff is we moved from aether_producer.py to __init_.py which already existed to make the name spacing of the module work more cleanly. No substantive changes were made to producer function.